common/src/test/java/gov/cms/ab2d/common/util/AB2DPostgresqlContainer.java

```diff
@@ -0,0 +1,34 @@
+package gov.cms.ab2d.common.util;
+
+import org.testcontainers.containers.PostgreSQLContainer;
+
+public class AB2DPostgresqlContainer extends PostgreSQLContainer<AB2DPostgresqlContainer> {
+
+    private static final String IMAGE_VERSION = "postgres:11";
+
+    private static AB2DPostgresqlContainer container;
+
+    private AB2DPostgresqlContainer() {
+        super(IMAGE_VERSION);
+    }
+
+    public static AB2DPostgresqlContainer getInstance() {
+        if (container == null) {
+            container = new AB2DPostgresqlContainer();
+        }
+        return container;
+    }
```

Comment:
Do we need to reuse the container across tests? If we use `new AB2DPostgresqlContainer()` instead in the tests, we will make sure each test starts with a clean DB.